### PR TITLE
PAYM-2815 Allow to set version for events

### PR DESCRIPTION
**Pull request by Kitton**
#### :tophat: Problem
We need to be able to set events version 
#### :pushpin: Solution
Add this feature

### CODE REVIEW

File: lib/events/event_handler.ex [hunk 1]
```diff
@@ -43,6 +43,9 @@ defmodule ElixirTools.Events.EventHandler do
     }
   end
 
+  @spec set_version(Event.t(), String.t()) :: Event.t()
```
Comment (hylkealons) [inline lib/events/event_handler.ex]: why not passing it through the `create_optional` param in `create/4`?

File: lib/events/event_handler.ex [hunk 2]
```diff
@@ -19,27 +19,29 @@ defmodule ElixirTools.Events.EventHandler do
...
-  @typep event_id_seed_optional :: String.t()
 
   @spec create(event_name, payload, event_id_seed) :: Event.t()
   def create(event_name, payload, event_id_seed) do
     %Event{name: event_name, payload: payload, event_id_seed: event_id_seed}
   end
 
   @typep create_optional ::
-           {:event_id_seed_optional, event_id_seed_optional}
+           {:event_id_seed_optional, String.t()}
```
Comment (dabeat) [inline lib/events/event_handler.ex]: What's the point of this change?

**Comment by dabeat** [comment]
:rocket:

File: lib/events/event_handler.ex (hunk 2 above)
Comment (Kitton) [reply lib/events/event_handler.ex]: remove a useless defined type

**Pull request merged**